data/help.txt

```diff
@@ -65,9 +69,26 @@ help "lost 7"
 help "map"
 	`This map shows all the star systems you know about and the hyperspace links between them. You can use the map to plot a hyperspace course by clicking on the system you want to travel to. To initiate a jump to that system, close the map and press <Initiate hyperspace jump>.`
 
+help "map advanced"
+	`As you explore more of the galaxy, the star map will quickly become a very powerful tool. Clicking on a distant system will automatically plot the quickest route to it. You can shift-click on systems to plot a route between them, or control-click to select it there setting a route.`
```

Comment:
Looks like I accidentally dropped the wrong word. Adding a bit more though just to flesh out the description.
```suggestion
	`As you explore more of the galaxy, the star map will quickly become a very powerful tool. Clicking on a distant system will automatically plot the quickest route to it. You can shift-click on multiple systems to plot a route between them, or control-click to select a system without setting a route to it.`
```